Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -109,11 +109,18 @@ extension SPAppDelegate {
     ///
     @objc
     func handleOpenTagList(url: NSURL) -> Bool {
-        guard let tag = url.internalTagKey else {
+        guard url.isInternalTagURL else {
             return false
         }
 
-        selectedTag = SPObjectManager.shared().tagExists(tag) ? tag : nil
+        if let tag = url.internalTagKey {
+            switch tag {
+            case String():
+                selectedTag = nil
```

Comment:
Why do we check for empty string here? 🤔

Reply:
Was doing that so that it returned something we could return for all notes that was not a specific string.  But this can be nil, I think I was using nil as a different check before.  I updated it